test/equal_test.c

```diff
@@ -0,0 +1,528 @@
+/*
+ * Copyright (c) 2014-2019 Pavel Kalvoda <me@pavelkalvoda.com>
+ *                    2019 Nicolas Mora <mail@babelouest.org>
+ *
+ * libcbor is free software; you can redistribute it and/or modify
+ * it under the terms of the MIT license. See LICENSE for details.
+ */
+
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+
+#include <cmocka.h>
+
+#include "assertions.h"
+#include "cbor.h"
+
+cbor_item_t *item1, *item2;
+
+static void test_equal_uint(void **state) {
```

Comment:
Please refactor this to be structured as one set of localized assertions per testcase

Reply:
do you mean one big `static void test_equal(void **state)` ?

Reply:
My bad, you mean the opposite, one function per test case, so more functions!

Reply:
done